core/src/main/resources/hudson/model/View/sidepanel.jelly

```diff
@@ -43,6 +46,47 @@ THE SOFTWARE.
         <l:task href="${rootURL}/${it.viewUrl}newJob" icon="icon-new-package icon-md" it="${app}" permission="${it.itemCreatePermission}" title="${%NewJob(it.newPronoun)}"/>
       </st:include>
 
+      <l:task href="#modal" icon="icon-new-package icon-md" title="Remodal Popup"/>
+      <!-- <st:include page="popup.jelly" optional="true" /> -->
+      
+      <div class="remodal new-job-popup" data-remodal-id="modal" id="add-item-panel">
```

Comment:
@lanwen is this better now? I successfully removed bootstrap components (container, row, etc.) and made the popup work without it. You can also see that I changed styles. And now the only rule I need is for .new-job-popup